kiwi/bootloader/config/zipl.py

```diff
@@ -149,10 +128,8 @@ def setup_disk_image_config(
             'title': self.quote_title(self.get_menu_entry_title()),
             'kernel_file': kernel,
             'initrd_file': initrd,
-            'boot_options': ' '.join([self.cmdline, boot_options]),
-            'failsafe_boot_options': ' '.join(
-                [self.cmdline_failsafe, boot_options]
-            )
+            'boot_options': self.cmdline,
```

Comment:
Is it mandatory to remove the boot options from the command line? I am surprised by this change. `boot_options` parameter is not used within the whole method. I assume this is on purpose, but just want to make sure this is not a simple mistake.

Reply:
Yes this was an intentional change. boot_options in this method is a dictionary containing information to setup and configure the bootloader. It should not be mixed with commandline options. For those we have other methods in that class. I realized the parameter name is not the best but don't want to change the api interface incompatible. Thus I updated the doc string which I hope makes the scope clear.

In the zipl case no further booting options like the efi device or things like that are needed. The code just creates the zipl config file as a static information